tensorflow_probability/python/mcmc/transformed_kernel.py

```diff
@@ -109,6 +105,35 @@ def transformed_log_prob_fn(*state_parts):
     return tlp + sum(ldj_fn(state_parts, event_ndims))
   return transformed_log_prob_fn
 
+def _make_kernel_stack(kernel):
+  kernel_stack = [kernel]
+  while 'target_log_prob_fn' not in kernel.parameters:
+    if 'inner_kernel' not in kernel.parameters:
+      raise ValueError('"None of the nested `inner_kernel`s contains a '
```

Comment:
this error message does not seem right, based on the conditions being checked.

Reply:
The idea is that if there are no more `inner_kernel`s we've hit the inner-most kernel and have already checked to see if it has a `target_log_prob_fn`. I did make the implicit assumption that all kernels will have an `inner_kernel` field; please correct me if I'm wrong